requirements/travis.txt

```diff
@@ -4,34 +4,24 @@
 #
 #    make upgrade
 #
-certifi==2019.9.11        # via requests, urllib3
-cffi==1.13.1              # via cryptography
+certifi==2019.9.11        # via requests
 chardet==3.0.4            # via requests
-configparser==4.0.2       # via importlib-metadata
-contextlib2==0.6.0.post1  # via importlib-metadata
 coverage==4.5.4           # via coveralls
 coveralls==1.8.2
-cryptography==2.8         # via pyopenssl, urllib3
 docopt==0.6.2             # via coveralls
-enum34==1.1.6             # via cryptography
 filelock==3.0.12          # via tox
-idna==2.8                 # via requests, urllib3
+idna==2.8                 # via requests
 importlib-metadata==0.23  # via pluggy, tox
-ipaddress==1.0.23         # via cryptography, urllib3
 more-itertools==5.0.0     # via zipp
 packaging==19.2           # via tox
-pathlib2==2.3.5           # via importlib-metadata
```

Comment:
It looks like `make upgrade` was run under Python 3, the Python 2 specific backports have been removed.

Reply:
Fixed.